src/Renderer/OBB.js

```diff
@@ -70,8 +71,14 @@ class OBB extends THREE.Object3D {
      */
     updateZ(min, max, scale = this.z.scale) {
         this.z = { min, max, scale, delta: Math.abs(max - min) * scale };
-        this.box3D.min.z = this.natBox.min.z + min * scale;
-        this.box3D.max.z = this.natBox.max.z + max * scale;
+        this.box3D.min.z = this.natBox.min.z + min * scale + this.geoidHeight;
+        this.box3D.max.z = this.natBox.max.z + max * scale + this.geoidHeight;
+    }
+
+    updateGeoid(geoidHeight) {
+        this.geoidHeight = geoidHeight;
+        this.box3D.min.z = this.natBox.min.z + this.z.min * this.z.scale + this.geoidHeight;
+        this.box3D.max.z = this.natBox.max.z + this.z.max * this.z.scale + this.geoidHeight;
```

Comment:
is necessary to have 2 methods?

Reply:
I factorize three methods of `OBB` (those two methods plus the `updateScale` method).